src/styles/blocks/grid.scss

```diff
@@ -0,0 +1,12 @@
+.container {
+  display: grid;
+  padding: 50px 40px;
+  grid-template-columns: repeat(4, 1fr);
+  max-width: 1024px;
+  margin: 0 auto;
+
+  .card:not(:last-child) {
+    margin-right: 48px;
+    margin-bottom: 46px;
+  }
+}
```

Comment:
https://github.com/mate-academy/layout_catalog/blob/master/grid.md Consider implementing all requirements for the grid layout